For backwards compatability gfortran has an intrinsic iargc so need to use that

diff -rupN want-2.5.1_orig//src/tools/decay.f90 want-2.5.1/src/tools/decay.f90
--- want-2.5.1_orig//src/tools/decay.f90	2014-05-24 14:13:05.000000000 +0200
+++ want-2.5.1/src/tools/decay.f90	2016-11-16 08:53:22.575769834 +0100
@@ -77,7 +77,7 @@
    REAL(dbl),     ALLOCATABLE :: decay(:)
    COMPLEX(dbl),  ALLOCATABLE :: opr(:,:,:), opr_aux(:,:,:)
    !
-   INTEGER, EXTERNAL :: iargc
+   INTEGER, INTRINSIC :: iargc
    !
    ! end of declariations
    !
